get.go

```diff
@@ -519,6 +519,23 @@ func (s Style) getBorderStyle() Border {
 	return s.borderStyle
 }
 
+// Returns whether or not the style has implicit borders. This happens when
+// a border style has been set but no border sides have been explicitly turned
+// on or off.
+func (s Style) implicitBorders() bool {
+	var (
+		borderStyle = s.getBorderStyle()
+		topSet      = s.isSet(borderTopKey)
+		rightSet    = s.isSet(borderRightKey)
+		bottomSet   = s.isSet(borderBottomKey)
+		leftSet     = s.isSet(borderLeftKey)
+	)
+	if borderStyle != noBorder && !(topSet || rightSet || bottomSet || leftSet) {
```

Comment:
I think you can remove this `if` condition and just return
```suggestion
	return borderStyle != noBorder && !(topSet || rightSet || bottomSet || leftSet)
```

Reply:
Good catch!